### DISCUSSION TITLE
Anybody using with RTL8761B ?

### DISCUSSION THREAD

**Discussion by Nuts-Underline** [emericg/toolBLEx#25] [Q&A]
Having RTL8761BUV (_0bda:a725_) based stick with external antenna, but it not working well, and with toolBLEx to: bad detecting of BLE devices.
Tested with Windows and some Linux distros, with same result. 
Same time `bluetoothctl ` is working, `scan le ` can detect some of my BLE devices, but need to play with 'power' options.
So i'd like to know is it toolBLEx issuse or broken BT adapter.

My older ASUS BT-400 (based BCM20702A0) working great with same PCs configs.

**Comment by Nuts-Underline** [comment]
updated to [toolBLEx 0.13] and instaled newest bluez from git, working now with RTL8761BUV (0bda:a725) but need to run `bluetoothctl power off` `bluetoothctl power on ` in terminal
also tryed other  RTL8761B (0bda:8771) stick, seems like it lesss stable

**Reply by emericg** [reply]
Glad it's working now, but there is no driver embedded in the app, so 0.12 and 0.13 are equivalent regarding device support.
I only use ASUS BT-400 and BT-500 on my computer, I think I have a RTL8761*something, but last time I tried it wasn't working on linux...

**Reply by Nuts-Underline** [reply]
ASUS USB-BT500 is RTL8761BU based, it's a tricky chip, but working, somehow, for example: https://www.home-assistant.io/integrations/bluetooth/#known-working-adapters
